opentelekomcloud/services/vpn/resource_opentelekomcloud_vpnaas_ipsec_policy_v2.go

```diff
@@ -36,28 +38,38 @@ func ResourceVpnIPSecPolicyV2() *schema.Resource {
 				ForceNew: true,
 			},
 			"name": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: common.ValidateName,
 			},
 			"auth_algorithm": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				Default:  "sha1",
+				ValidateFunc: validation.StringInSlice([]string{
+					"md5", "sha1", "sha2-256", "sha2-384", "sha2-512",
+				}, false),
 			},
 			"encapsulation_mode": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				Default:  "tunnel",
 			},
 			"pfs": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				Default:  "group5",
+				ValidateFunc: validation.StringInSlice([]string{
+					"group1", "group2", "group5", "group14", "group15", "group16", "group19", "group20", "group21",
```

Comment:
List of groups here and in the doc differs

Reply:
updated